src/applications/vaos/utils/events.js

```diff
@@ -93,3 +94,33 @@ export function recordAppointmentDetailsNullStates(attributes, nullStates) {
     'fields-load-fail': missingFields.join(','),
   });
 }
+
+export function captureMissingModalityLogs(appointment) {
+  Sentry.withScope(scope => {
```

Comment:
@eselkin is there a way to test this somehow? I'm a little paranoid that this could break things so I'm trying to be extra careful. I tested the payload against mock data and it looked fine but I'm not sure what else I can do to further avoid risking prod.